### ISSUE TITLE
File modification time updates even if autoimport does not change file

### ISSUE THREAD

**Issue by Jasha10** [lyz-code/autoimport#179]
## Feature Request
If I run autoimport on a file and the file is not changed, the modification time of the file should not be changed either.

For example:
```
$ echo "print(123)" > foo.py
$ cat foo.py
print(123)
$ date -r foo.py
Thu Jan 20 17:37:42 CST 2022
$ autoimport foo.py
$ cat foo.py
print(123)
$ # ^ foo.py was not changed
$ date -r foo.py
Thu Jan 20 17:37:51 CST 2022
$ # ^ file modification time has changed!
```
`autoimport` did not modify `foo.py`, but the modification time was changed.

## Possible Solution
Perhaps autoimport could check whether the file would be modified, and only touch the file if a nonzero diff would occur.

## Additional context
I use makefiles and other similar tools which keep track of file modification time. If autoimport does not change the file's contents, modification of the file modification time triggers a false-positive make/build cycle.

**Comment by lyz-code** [comment]
Hi @Jasha10 thanks again for opening the issue, I'll look into it next week, as always, if you can make a PR I'll be grateful

**Comment by lyz-code** [comment]
Fixed in https://github.com/lyz-code/autoimport/pull/181 and released under `1.0.2`, plz test if it works for you
